roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -168,13 +168,22 @@ istio_enabled: false
 registry_enabled: false
 enable_network_policy: false
 local_volume_provisioner_enabled: false
+cephfs_provisioner_enabled: false
 persistent_volumes_enabled: false
 
 # Base path for local volume provisioner addon
 local_volume_base_dir: /mnt/disks
 local_volume_mount_dir: /local-disks
 local_volume_storage_class: local-storage
 
+# Defaults for CephFS provisioner addon
+cephfs_provisioner_namespace: "{{ system_namespace }}"
```

Comment:
don't need to put those here. 
Eventually, add them to the `group_vars/k8s-cluster.yaml` in comment so user can discover the configuration more easily